React/Base/RCTConvert.m

```diff
@@ -729,7 +729,22 @@ +(type)type : (id)json                                      \
         // iOS 13.0
         RCTFallbackARGB : @(0xFFf2f2f7)
       },
+    } mutableCopy];
+    // The color names are the Objective-C UIColor selector names,
+    // but Swift selector names are valid as well, so make aliases.
+    static NSString *const RCTColorSuffix = @"Color";
+    NSMutableDictionary<NSString *, NSDictionary *> *aliases = [NSMutableDictionary new];
+    for (NSString *objcSelector in map) {
+      RCTAssert([objcSelector hasSuffix:RCTColorSuffix], @"A selector in the color map did not end with the suffix Color.");
+      NSMutableDictionary *entry = [map[objcSelector] mutableCopy];
+      RCTAssert([entry objectForKey:RCTSelector] == nil, @"Entry should not already have an RCTSelector");
+      NSString *swiftSelector = [objcSelector substringToIndex:[objcSelector length] - [RCTColorSuffix length]];
```

Comment:
Seems like this code could have been slightly simpler/faster if it had been reversed, i.e. manually add them all without the `Color` suffix and then alias them by appending the suffix. Not a huge deal [on its own], though, as this map is cached.

Reply:
If reversed, it would mean having to add an explicit RCTSelector for each entry -- which is more error prone for maintenance.   I like it this way because the table is more truthy to ObjC which is the native language of the RN implementation itself.